external/bootstrap.cmake

```diff
@@ -47,36 +49,15 @@ execute_process(COMMAND ${CMAKE_COMMAND}
 if(result)
     message(FATAL_ERROR "CMake step for Third Party builds failed: ${result}")
 endif()
-if(MSVC)
-  # for MSVC builds we need to build both Release and Debug builds
```

Comment:
I think we should not do this (double Release, Debug build). The IDE should fix that, or one can call Clean and Rebuild.. 
This way, the CI build time is artificially doubled